Source/RimConnection/Actions/Events/RaidFriendly.cs

```diff
@@ -35,7 +35,7 @@ public override void Execute(int amount, string boughtBy)
             raidWorker.def = IncidentDef.Named("RaidFriendly");
             raidWorker.TryExecute(parms);
 
-            AlertManager.NormalEventNotification("Your viewers have sent help from ");
+            AlertManager.NormalEventNotification("{0} have sent help from ", boughtBy); //The message cuts off? TODO
```

Comment:
Yeah I think this is an unfinished event that isn't included in the big list of events that are usable. Couldn't figure it out but must have accidentally committed it

Reply:
Okay, I'll keep this as-is then.